### ISSUE TITLE
Filters List: bigger Tooltips or detailed help

### ISSUE THREAD

**Issue by GoeWink** [RealRaven2000/quickFilters#237]
I like Quickfilters very much and use intensively.
And I like the menue where I can merge and copy and so on, too. I see there are toolrips which pop up with mosue-over. That is fine, too.
But I am handicpped by view and have to magnify to read  little letters. Normally that works fine  but the longer the text the less I can read. Following the long text the mouse-over get lost and the text is gone.
So I ask for bigger letters. Or  a help text with piktogram to find the right actons.
Sure: Many pictograms are known but the others...?
Thank you in advance

**Comment by RealRaven2000** [comment]
I guess you mean the "icon only" toolbar in the filters list? I will look into it - these are determined by some built in CSS, based on the toolbarbutton XUL elements, so this is actually then a general accessibility issue that will affect any such tooltips anywhere else in the Thunderbird UI.

![image](https://github.com/RealRaven2000/quickFilters/assets/6314674/ffb0ab56-48bd-41fd-9702-0fe62d73e121)

So I asked the Thunderbird developers and the general Mozilla developers (Firefox) on Matrix chat. Apparently there is a single tooltip element at the end of the DOM document:

![image](https://github.com/RealRaven2000/quickFilters/assets/6314674/55d81be8-a80d-428b-b4ea-0ceb09391880)

so we should be able to style this with a CSS rule and increase its font size...

**Comment by RealRaven2000** [comment]
Original CSS (from user agent):

```css
tooltip {
  white-space: pre-wrap;
  color: InfoText;
  font: message-box;
  pointer-events: none;
}
```

I will override this with:

```css
tooltip {
 font-size: 120%;
}
```

However, that didn't work, I asked and:

![image](https://github.com/RealRaven2000/quickFilters/assets/6314674/a1c39f0d-0aee-49ab-ad30-ddd04099ce59)

however, there seems to be another solution:

![image](https://github.com/RealRaven2000/quickFilters/assets/6314674/41df4d88-c1d7-4d0c-b9d0-af9c4767b978)

So I basically need to implement my own personal tooltip element and style that. WE will see...

if that doesn't work I can try a CSS only solution that injects the text using a content pseudo element ::before oder ::after  (which I have done before in FiltaQuilla)

**Comment by RealRaven2000** [comment]
Ok, I came up with a solution - this is a custom tooltip with different positioning and larger font. So it also works well if you have large mouse cursors (preventing `[this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1712669)`). The trick is to not use the `tooltiptext` attribute - because then it will always use the default tooltip and then then set a `tooltip` attribute. The tooltip is it's on element and has a onpopupshowing event listener that extracts the custom tooltip text (I called it `tooltiptext_qI`)

![image](https://github.com/RealRaven2000/quickFilters/assets/6314674/e87daf03-d6ab-40b3-9b95-4cfa7f2a885e)

test version below (restart Thunderbird to force loading the new style sheet) 

[quickFilters-wx-6.4pre50.zip](https://github.com/RealRaven2000/quickFilters/files/14946565/quickFilters-wx-6.4pre50.zip)

----
To install version above download zip file and drag the file into Thunderbird Add-ons Manager (do not extract contents, it won't install like that)
